.github/workflows/coverage.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       - uses: actions/checkout@v2
       - name: Starting Gremlin Servers
         run: |
-          docker-compose -f ./docker-compose/docker-compose.yaml up -d 
```

Comment:
Seems `docker-compose` (note the middle `-` is no longer a known command on the latest action runners, so changed it here and in the test workflow files